partiql-lang/src/main/pig/partiql.ion

```diff
@@ -76,6 +76,9 @@ may then be further optimized by selecting better implementations of each operat
 
         // The expressions that can result in values.
         (sum expr
+            // SQL Session Keywords (CURRENT_USER, CURRENT_ROLE, etc.)
```

Comment:
I think it makes sense to disallow `CURRENT_USER` as a keyword (E.g. when used as `SELECT "CURRENT_USER"` ...).

All the following are error in PostgresSQL 15:

```
CREATE TABLE MyTable (
  current_user VARCHAR(20)
);

CREATE TABLE MyTable (
  CURRENT_USER VARCHAR(20)
);

CREATE TABLE CURRENT_USER (
  id VARCHAR(20) 
);

```

Ref. https://www.db-fiddle.com/f/dsYWqRGdH4UDoPSHV5wKfC/0

As also discussed internally—echoing my view and also @vgapeyev's—we can just parse directly to a `Call` node.